CONTRIBUTING.md

```diff
@@ -98,5 +96,3 @@ Describe what’s the pull request about in one short sentence and post the link
 * Please provide as much context as possible when you open an issue. The information you provide must be comprehensive enough to reproduce that issue for the assignee. Therefore, contributors should use but aren't restricted to the issue template provided by the project maintainers.
 
 * When creating an issue, try using one of our issue templates which already contain some guidelines on which content is expected to process the issue most efficiently. If no template applies, you can of course also create an issue from scratch.
-
-* Please apply one or more applicable [labels](https://github.com/corona-warn-app/cwa-app-ios/labels) to your issue so that all community members are able to cluster the issues better.
```

Comment:
I think this line should stay deleted since the most issues are opened by outside contributors who can't apply labels @marcussc